modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldValueRenderer.java

```diff
@@ -63,7 +63,18 @@ private String _render(Locale defaultLocale, String valueString) {
 			return StringPool.BLANK;
 		}
 
-		LocalDate localDate = DateParameterUtil.getLocalDate(valueString);
```

Comment:
@rodrigopaulino don't we need to change the `DateParameterUtil` too? I tried to use the Date validation with Arabic language and it didn’t work.

If it takes a lot of effort to do this (since we don't pass the locale in `getLocalDate`), it would be good to create a bug for the date validation use case.

Reply:
That's what I was thinking, @carolmariaabb. Since it requires other steps to reproduce, I thought it would be better if they opened another bug. Besides, I'd prefer to not proliferate this behavior, because this is just a workaround to combat the technical debt described [here](https://issues.liferay.com/browse/LPS-144731).

Reply:
Got it!